index.html

```diff
@@ -54,20 +54,20 @@
           <ul>
             <li>
               <a href="#"
-                ><img src="./assets/icons/github.png" alt="github icon"
+                ><img src="./assets/icons/github.png" alt="github"
```

Comment:
Kindly change the contrast to be at least `4.5` of the `see project` buttons because they failed in the contract check
![image](https://user-images.githubusercontent.com/24703171/155036938-747d7ca8-acd2-41a1-a92a-52f1fc2e0616.png)
You can check the contrast for your website using this [link](https://www.aditus.io/button-contrast-checker/)

Reply:
@gemmen29; so for the contrast, is it ok if I modify the Figma original color scheme?

<img width="875" alt="Screen Shot 2022-02-21 at 6 29 01 PM" src="https://user-images.githubusercontent.com/45613824/155038118-3f3db940-71e3-427e-ba3e-b8f2142f6ee9.png">

Reply:
Yes, but try to use similar color as much as possible